models/project_repository.go

```diff
@@ -0,0 +1,188 @@
+package models
```

Comment:
Right, but if interface and Impl are in "project", then there is no import from project->appl